### DISCUSSION TITLE
name(path) with special characters `'`

### DISCUSSION THREAD

**Discussion by sakalx** [react-hook-form/react-hook-form#9425] [General]
**Is your feature request related to a problem? Please describe.**
in `v5` we could use `name` with special characters 
for example: `"issuerRatings.MOODY'S"`, 
but in `v7` `getValues` `errors`  will not see that values
example: https://codesandbox.io/s/react-hook-form-v7-getvalues-forked-x4oxh8?file=/src/index.jsx

**Describe the solution you'd like**
similar behavior as we have in v5

**Describe alternatives you've considered**
🤷‍♂️

**Additional context**
Thanks !

**Comment by bluebill1049** [comment]
I would consider moving away from those special characters, as you never know which one is going to break the path lookup. What's the specific use case that you try to cover, is any other workaround to resolve your issue?

**Comment by bluebill1049** [comment]
Going to convert this to general discussion, feel free to discuss it there.
